src/state/perpetualsSelectors.ts

```diff
@@ -159,3 +159,10 @@ export const getCurrentMarketNextFundingRate = createSelector(
   [getCurrentMarketData],
   (marketData) => marketData?.perpetual?.nextFundingRate
 );
+
+/**
+ * @param marketId
+ * @returns sparkline data for specified marketId
+ */
+export const getPerpetualMarketSparklineData = (marketId: string) => (state: RootState) =>
```

Comment:
nit: Looks like we are not using this, and is instead using the hook for get it from indexer.

Reply:
@rosepuppy, This selector isn't being used for this feature but I think it should be okay leaving it as it accesses `Abacus/Market/Line` which will be used on the Markets page refresh.